test/qa/lib/firewall.py

```diff
@@ -92,99 +95,231 @@
     "-A OUTPUT -d 10.0.0.0/8 -o eth0 -m comment --comment nordvpn -j ACCEPT",
 ]
 
+
+def _get_rules_killswitch_on(interface:str):
+    return \
+    [
+        f"-A INPUT -i {interface} -m connmark --mark 0xe1f1 -m comment --comment nordvpn -j ACCEPT",
+        f"-A INPUT -i {interface} -m comment --comment nordvpn -j DROP",
+        f"-A OUTPUT -o {interface} -m mark --mark 0xe1f1 -m comment --comment nordvpn -j CONNMARK --save-mark --nfmask 0xffffffff --ctmask 0xffffffff",
+        f"-A OUTPUT -o {interface} -m connmark --mark 0xe1f1 -m comment --comment nordvpn -j ACCEPT",
+        f"-A OUTPUT -o {interface} -m comment --comment nordvpn -j DROP"
+    ]
+
+
+def _get_rules_connected_to_vpn_server(interface:str):
```

Comment:
Minor point, but what is the point of having function which adds nothing? Maybe just use original function?